README.md

```diff
@@ -28,6 +29,7 @@ module "default" {
 | `generate_ssh_key`           | `false`        | If set to `true`, new ssh key pair will be created      | No       |
 | `private_key_extension`      | ``             | Private key file extension, _e.g._ `.pem`               | No       |
 | `public_key_extension`       | `.pub`         | Public key file extension, _e.g._ `.pub`                | Yes      |
+| `chmod_command`              | `chmod 600 %v` | The command executed on the private key file            | No       |
```

Comment:
better to call it `Template of the command executed on the private key file.  Required on Linux. Optional on Windows`

Reply:
yep, fix it